integration_tests/tests/temporary-accommodation/manage/lostBed.cy.ts

```diff
@@ -163,9 +160,7 @@ context('Lost bed', () => {
 
     // And I fill out the form with dates that conflict with an existing lost bed
     const lostBed = lostBedFactory.build({
-      bed: bedFactory.build({
-        id: room.id,
-      }),
+      bedId: room.id,
```

Comment:
Should this and the below change be `bedId: room.beds[0].id`?

Reply:
yes, good spot